cmd/yace/main.go

```diff
@@ -341,7 +340,7 @@ func startScraper(c *cli.Context) error {
 		go s.decoupled(ctx, logger, newJobsCfg, cache)
 	})
 
-	logger.Info("Yace startup completed", "version", version, "feature_flags", strings.Join(featureFlags, ","))
+	logger.Info("Yace startup completed", "build_context", version.BuildContext(), "feature_flags", strings.Join(featureFlags, ","))
```

Comment:
Just double-checking, did you intend to put the whole buildcontext as a single value? The output looks like:

```
"build_context":"(go=go1.23.4, platform=darwin/arm64, user=cristian@MacG2.local, date=20250107-08:53:15, tags=unknown)"
```

Reply:
Yes, that's typically how we've done it in other exporters.

For example:
```
time=2025-01-07T13:54:03.868+01:00 level=INFO source=/home/ben/src/prometheus-community/bind_exporter/bind_exporter.go:559 msg="Starting bind_exporter" version="(version=0.7.0, branch=superq/slog, revision=ea563db0a26ecc5c32bd36f0770ca81ef2c8be9c)"
```

Reply:
Maybe we could add `version.Info()` in this very log line next to the build context, so at least the branch and revision are outputted somehow.

Reply:
```json
{"time":"2025-01-08T17:36:17.014336719Z","level":"INFO","source":"main.go:343","msg":"Yace startup completed","version":"0.62.1","version":"0.62.1","build_context":"(go=go1.23.3, platform=linux/amd64, user=ben@mrhat, date=20250108-17:36:06, tags=unknown)","feature_flags":""}
```